test-models/github-model.test.js

```
@@ -0,0 +1,204 @@
+      console.log("=== Response from GitHub Model ===");
+      console.log(response);
+      
+      assert.ok(response, 'Should receive a non-empty response');
+      
+      // Try to parse the response as JSON
+      try {
+        const parsedResponse = JSON.parse(response);
+        
+        // Validate response structure
+        assert.ok(
+          parsedResponse.hasOwnProperty('urgency') && 
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)